src/ArduinoCellular.cpp

```diff
@@ -122,9 +122,16 @@ Time ArduinoCellular::getGPSTime(){
 }
 
 Time ArduinoCellular::getCellularTime(){
-    int year, month, day, hour, minute, second;
+    int year = 1970;
+    int month = 1;
+    int day = 1;
+    int hour = 0;
+    int minute = 0;
+    int second = 0;
     float tz;
-    modem.getNetworkTime(&year, &month, &day, &hour, &minute, &second, &tz);
+    if (modem.NTPServerSync()) {
```

Comment:
```suggestion
    if (modem.NTPServerSync() == 0) {
```
I think a return code of 0 indicates success. @cristidragomir97 can you double check?

Reply:
Oh yes, i was for sure wrong. I've changed it checking if `!= -1`  to be on the safe side :slightly_smiling_face:

Reply:
@pennam I think that -1 is the generic error code while there are other `+CNTP: ` result codes that indicate an error. I'd check for == 0.

Reply:
@sebromero Thanks! I was wrong again :smiling_face_with_tear: